.github/actions/calculate-stats-action/main.js

```diff
@@ -12,6 +12,16 @@ try {
     core.setOutput("stats-output", status);
 
 } catch(error) {
-    core.setFailed(error.message);
-    console.log(`Error parsing JSON string: ${error}`);
+    try {
+        const testPasses = content.numPassedTests;
```

Comment:
I don't think you can nest try...catch @TundeOladejo

Reply:
It's possible to nest a `try..catch` statement.